torch/onnx/utils.py

```diff
@@ -316,6 +316,18 @@ def _decide_external_data_format(use_external_data_format, operator_export_type,
     model_file_location = f if val_use_external_data_format and isinstance(f, str) else str()
     return val_use_external_data_format, model_file_location
 
+def _decide_input_format(model, args):
+    import inspect
+    sig = inspect.signature(model.forward)
+    ordered_list_keys = list(sig.parameters.keys())
+    if isinstance(args[-1], dict):
```

Comment:
This assumes that the dictionary input must come at last, shall we document it somewhere?

Reply:
Adding documentation